database/controllers/dbm.js

```diff
@@ -48,8 +47,34 @@ module.exports = {
     //Description:  This function will return all User data from
     // the given start date to the given end date.  If start date
     // is undefined then all users will be returned.
-    getUser(from, to){
-        return true;
+    getUsers(from, to){
+        if(from === undefined){
```

Comment:
Doesnt this need to check if both `from` and `to` are undefined? and then check if both are defined. And if one or the other are defined but not both throw an error?

Reply:
I could definitely do it that way.

Reply:
Updated to check if both are undefined now.  If one is undefined but not the other the sql query in the else statement will throw an error, otherwise it will return the results based off the two entered dates.